hie.yaml

```diff
@@ -1,14 +1,18 @@
 cradle:
-  stack:
-    - path: "./src"
-      component: "hoff:lib"
-    - path: "./app"
+  cabal:
+    - path: "src"
+      component: "lib:hoff"
+    - path: "app/Main.hs"
       component: "hoff:exe:hoff"
-    - path: "./tests/EndToEnd.hs"
+    - path: "tests"
+      component: "hoff:test:spec"
+
+      # These need to be specified manually since they're in the same directory.
```

Comment:
Thank you! I'm not sure how you came across this but thanks for setting it up \o/